config/locales/en.yml

```diff
@@ -575,9 +578,12 @@ en:
   # /gov-one/info
   gov_one_info:
     title: Gov One Info
-    body: This is some information about Gov One Login.
+    hero_header: How to access this training course
```

Comment:
Because I18n has `:scope` I like to nest keys where possible. Here that would create 2 keys under `gov_one_info.hero`